pyscriptjs/src/pyodide.ts

```diff
@@ -70,8 +70,8 @@ export class PyodideRuntime extends Runtime {
         logger.info('pyodide loaded and initialized');
     }
 
-    async run(code: string): Promise<any> {
-        return await this.interpreter.runPythonAsync(code);
+    run(code: string): Promise<any> {
+        return this.interpreter.runPython(code);
```

Comment:
I'm confused: why does it still return a promise?

Reply:
An oversight on my part - fixed!